.travis.yml

```diff
@@ -33,6 +33,9 @@ android:
     - 'android-sdk-license-.+'
     - 'google-gdk-license-.+'
 
+before_install:
+  - yes | sdkmanager "platforms;android-30"
+  - yes | sdkmanager "build-tools;29.0.2"
```

Comment:
The Travis build failed with an error that licenses for `platforms;android-30` and `build-tools;29.0.2` were not accepted. [The Travis Android docs](https://docs.travis-ci.com/user/languages/android/#dealing-with-licenses) said that licenses are automatically accepted but tweaking the `licenses` section of the `.travis.yml` didn't work. Adding this `before_install` step seems to do the trick.